src/Main.purs

```diff
@@ -223,15 +223,17 @@ succeeded :: Unit -> EffN Unit
 succeeded = const (pure unit)
 
 main :: EffN Unit
-main = runAff failed succeeded do
-  requireNodeAtLeast (version 4 0 0 Nil Nil)
-  argv <- drop 2 <$> liftEff Process.argv
-  args <- parse globals commands argv
-  case args of
-    Left (ParseError { message: err }) ->
-      handleParseError (head argv) err
-    Right args' -> do
-      runArgs args'
+main = do 
+  runAff failed succeeded do
+    requireNodeAtLeast (version 4 0 0 Nil Nil)
+    argv <- drop 2 <$> liftEff Process.argv
+    args <- parse globals commands argv
+    case args of
+      Left (ParseError { message: err }) ->
+        handleParseError (head argv) err
+      Right args' -> do
+        runArgs args'
+  pure unit
```

Comment:
^ this is the other place where I think we should use `void`, and now that the Aff changes I mentioned earlier have been dealt with I think we can use `void` here.